talentup/webapp-revamp
# Setup sass and redux

### What does this PR do?
- setup redux and sass
- integrated redux state to personal info form

### How can it be manually tested?
- launch app
- click on edit profile
- edit profile details, it should be updated everywhere else
